source/guide/integrate-to-your-app.md

```diff
@@ -8,102 +8,50 @@ version: 2.1
 
 # Integrate to Your App
 
-## Integrate to Android
-Tip：The following documents assume that you already have a certain Android development experience.
+## Integrate to Android Platform
+The following documents assume that you already have a certain Android development experience.
 
-### Android has two ways to integrate weex
-1.using source code: Can quickly use the latest features of WEEX, according to your own characteristics of the project. So, you can do some related improvements.
 
-2.using SDK: WEEX will regularly release a stable version at jcenter.[jcenter](https://bintray.com/alibabaweex/maven/weex_sdk/view)
+### Quick Start Five Steps
 
-### Prerequisites
-Make sure the following configuration is complete:
+The keys to intergrating Weex into your Android application are the following five step:
```

Comment:
Five quick steps to start profiling intergrating Weex into your Android application: